packages/protocol/contracts/governance/Governance.sol

```diff
@@ -200,9 +200,15 @@ contract Governance is
    * @return The storage, major, minor, and patch version of the contract.
    */
   function getVersionNumber() external pure returns (uint256, uint256, uint256, uint256) {
-    return (1, 2, 0, 3);
+    return (1, 2, 1, 2);
```

Comment:
```suggestion
    return (1, 2, 1, 0);
```
the minor bump supercedes any patch change

Reply:
I'm actually not sure why this passed in CI...
@nambrot

Reply:
return (1, 2, 1, 0) is correct right? When minor is updated patch becomes 0. This is as you suggested in your previous message. I had to make multiple commits to fix this.

Reply:
yes, thank you!

Reply:
@yorhodes sounds like a ticket to me :P